flathub.json

```diff
@@ -0,0 +1,3 @@
+{
+    "only-arches": ["x86_64"]
```

Comment:
:-(

Reply:
What should I do to resolve this? It's very vague, I'm new to this and want to learn what I'm doing wrong.

Reply:
have a build for aarch64.

It's my disappointement that once again aarch64 users are left out. For software for a libre license.

Reply:
> have a build for aarch64.
> 
> It's my disappointement that once again aarch64 users are left out. For software with a libre license.

Ok I will forward this to upstream and also look into creating a version for aarch64 myself.

Reply:
So according to upstream, widevine doesn't support aarch64 on Linux.